lametro/models.py

```diff
@@ -391,7 +391,12 @@ def headshot_url(self):
             filename
         )
 
-        if Path(manual_headshot).exists():
+        if self.image:
+            # Assigning this to image_url would make the return static() at the
+            # end of this add 'static' to the url. Returning here solves that.
+            return self.image
```

Comment:
Nice! How about returning image, if it exists, at the very top of this property – that way we aren't doing any of the other work if we don't need to?

Reply:
Oh, yea this makes sense! We wouldn't need any of the file directory business if the image just already exists.